### PR TITLE
Create 2021-02-11-pif-18f-usds.md

### PR DESCRIPTION
**Pull request by wslack**

### CODE REVIEW

File: _posts/2021-02-11-pif-18f-usds.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+title: Comparing USDS, 18F, and PIF
+author: Will Slack
+layout: post
+permalink: /pif-18f-usds/
+---
+
+This post seeks to contrast USDS, 18F, and the Presidential Innovation Fellows (PIF) program using public information, leveraging my experience working at 18F alongside PIFs and later at USDS. I won't cover the the many wonderful shared aspects of the programs, but briefly, all three have staff appointed to term-limited positions, share a common goal of improving federal government tech capacity, and seek to elevate and empower career civil servant partners.
```
Comment (mheadd) [inline _posts/2021-02-11-pif-18f-usds.md]: double `the` in second sentence.

File: _posts/2021-02-11-pif-18f-usds.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+I should also mention two organizations doing related digital work: [Technology Transformation Services (TTS) Solutions](https://www.gsa.gov/about-us/organization/federal-acquisition-service/technology-transformation-services/tts-solutions), which manages a variety of programs such as [usa.gov](https://usa.gov), [cloud.gov](https://cloud.gov), [data.gov](https://data.gov), and [login.gov](https://login.gov), is mainly composed of career civil servants, and the [TTS Centers of Excellence](https://coe.gsa.gov/), which is set up in portfolios like AI and IT Infrastructure Optimization. All of these organizations changed under both Obama and Trump, and while I have no inside information, it’s likely there will be more change in this administration. If you’re interested in being part of the future of any of these organizations, please apply!
```
Comment (mheadd) [inline _posts/2021-02-11-pif-18f-usds.md]: Wonder if it makes sense to call out DDS as well in this section.

File: _posts/2021-02-11-pif-18f-usds.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+### Parent agency
+
+USDS is based in the [Executive Office of the President](https://www.whitehouse.gov/administration/executive-office-of-the-president/), specifically in the [Office of Management and Budget (OMB)](https://www.whitehouse.gov/omb/), which oversees executive branch agencies and assembles the yearly budget request for Congress’s consideration. Being part of "The White House" can make USDS seem intimidating, though it can also open doors. USDS's office is in a [converted 1870s town house](https://www.federaltimes.com/it-networks/2018/07/25/inside-the-agency-where-you-wish-you-worked/) once briefly occupied by President Theodore Roosevelt.
+
+18F and PIF are based in the General Services Administration, a support agency for the executive branch (for example, it manages most federal real estate and manages large contract vehicles). 18F and PIF have hoteled office space in GSA's headquarters a block from the White House at 1800 F street (the [origin of 18F's name](https://18f.gsa.gov/about/#history-and-funding)).
```
Comment (mheadd) [inline _posts/2021-02-11-pif-18f-usds.md]: Capitalize `Street`

File: _posts/2021-02-11-pif-18f-usds.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+### Engagement incubation
+
+USDS’s appropriation allows it to jump into supporting an agency on a problem immediately, though work often starts with a [discovery sprint](https://www.linkedin.com/pulse/government-discovery-sprint-playbook-how-lessons-learned-kathy-pham/). 18F sets up a formal agreement before working on an engagement, which requires signoff from multiple partner officials and in my experience, usually took over a month to execute. PIFs are deployed to agencies annually, and agencies submit projects for PIFs months in advance to be considered for that year’s class of fellows under a similar agreement to 18F's. It’s also common for PIFs to flex to other special projects once they start at an agency, based on needs that come up throughout the year.
```
Comment (mheadd) [inline _posts/2021-02-11-pif-18f-usds.md]: Worth adding a link to 18F's `How we work` in this section? https://18f.gsa.gov/how-we-work/

File: _posts/2021-02-11-pif-18f-usds.md (hunk 1 above)
Comment (wslack) [reply _posts/2021-02-11-pif-18f-usds.md]:
```suggestion
This post seeks to contrast USDS, 18F, and the Presidential Innovation Fellows (PIF) program using public information, leveraging my experience working at 18F alongside PIFs and later at USDS. I won't cover the many wonderful shared aspects of the programs, but briefly, all three have staff appointed to term-limited positions, share a common goal of improving federal government tech capacity, and seek to elevate and empower career civil servant partners.
```

File: _posts/2021-02-11-pif-18f-usds.md (hunk 3 above)
Comment (wslack) [inline _posts/2021-02-11-pif-18f-usds.md]:
```suggestion
18F and PIF are based in the General Services Administration, a support agency for the executive branch (for example, it manages most federal real estate and manages large contract vehicles). 18F and PIF have hoteled office space in GSA's headquarters a block from the White House at 1800 F Street (the [origin of 18F's name](https://18f.gsa.gov/about/#history-and-funding)).
```

File: _posts/2021-02-11-pif-18f-usds.md (hunk 4 above)
Comment (wslack) [reply _posts/2021-02-11-pif-18f-usds.md]: great call!

File: _posts/2021-02-11-pif-18f-usds.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
```
Comment (wslack) [inline _posts/2021-02-11-pif-18f-usds.md]:
```suggestion
USDS’s appropriation allows it to jump into supporting an agency on a problem immediately, though work often starts with a [discovery sprint](https://www.linkedin.com/pulse/government-discovery-sprint-playbook-how-lessons-learned-kathy-pham/). 18F [sets up a formal agreement](https://18f.gsa.gov/how-we-work/) before working on an engagement, which requires sign-off from multiple partner officials and in my experience, usually took over a month to execute. PIFs are deployed to agencies annually, and agencies submit projects for PIFs months in advance to be considered for that year’s class of fellows under a similar agreement to 18F's. It’s also common for PIFs to flex to other special projects once they start at an agency, based on needs that come up throughout the year.
```

File: _posts/2021-02-11-pif-18f-usds.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+
+I should also mention two organizations doing related digital work: [Technology Transformation Services (TTS) Solutions](https://www.gsa.gov/about-us/organization/federal-acquisition-service/technology-transformation-services/tts-solutions), which manages a variety of programs such as [usa.gov](https://usa.gov), [cloud.gov](https://cloud.gov), [data.gov](https://data.gov), and [login.gov](https://login.gov), is mainly composed of career civil servants, and the [TTS Centers of Excellence](https://coe.gsa.gov/), which is set up in portfolios like AI and IT Infrastructure Optimization. All of these organizations changed under both Obama and Trump, and while I have no inside information, it’s likely there will be more change in this administration. If you’re interested in being part of the future of any of these organizations, please apply!
```
Comment (wslack) [inline _posts/2021-02-11-pif-18f-usds.md]:
```suggestion
I should also mention two organizations doing related cross-gov digital work: [Technology Transformation Services (TTS) Solutions](https://www.gsa.gov/about-us/organization/federal-acquisition-service/technology-transformation-services/tts-solutions), which manages a variety of programs such as [usa.gov](https://usa.gov), [cloud.gov](https://cloud.gov), [data.gov](https://data.gov), and [login.gov](https://login.gov), is mainly composed of career civil servants, and the [TTS Centers of Excellence](https://coe.gsa.gov/), which is set up in portfolios like AI and IT Infrastructure Optimization. All of these organizations changed under both Obama and Trump, and while I have no inside information, it’s likely there will be more change in this administration. If you’re interested in being part of the future of any of these organizations, please apply! 
```

**Pull request merged**